app/controllers/concerns/locale_detection.rb

```diff
@@ -1,17 +1,19 @@
+# frozen_string_literal: true
+
 module LocaleDetection
   protected
 
   def switch_locale
     locale = params[:locale] || cookies[:locale]
-    if allowed_locale?(locale)
-      I18n.locale = locale
-    else
-      I18n.locale = default_locale
-    end
+    I18n.locale = if allowed_locale?(locale)
```

Comment:
a) this formatting is not great b) i think the rule is not great, i'd rather use a ternary operator

Reply:
Totally agree, I don't know how did rubocop autocorrect do this:D